_posts/2022-12-05-Introducting-Identity.markdown

```diff
@@ -0,0 +1,35 @@
+# Introducing Identity for OpenSearch
+
+OpenSearch handles critical data; how that data is permissioned and how a cluster is operated is a Big Deal™.  Consider the question "Who can delete all the indices in the cluster?", in the core of OpenSearch, the answer is whoever can make a request to the cluster.  
+
+Within the OpenSearch Project, there is a plugin Security that allows finer levels of access control providing a more satisfying answer.  Security uses the plugin architecture of OpenSearch that forces concepts around identity to be optional in core, plugins, and extensions components.
+
+Let’s permeate identity in every aspect of OpenSearch – enabling security by default - everywhere.
+
+## What are OpenSearch’s identities?
+In the literature, an identity is an Authenticated [1] Subject [2].  In the systems of OpenSearch, an identity embodies sources of interaction.

```

Comment:
Thoughts on `In OpenSearch systems, interaction with a cluster should be clearly associated with an identity. Sources of interaction include: ...`